.env.example

```diff
@@ -2,3 +2,4 @@ GOOGLE_API_KEY="Sample Key"
 API_HOST=https://open-event-api-dev.herokuapp.com
 MAP_DISPLAY=embed
 FASTBOOT_DISABLED=true
+WEB_APP_GENERATOR_HOST=https://open-event-wsgen-dev.herokuapp.com
```

Comment:
Since the default one is already specified in env and this is not a major component, please remove it from here